### PR TITLE
add tests for 1.6.4.1 and 1.6.4.2

### PR DESCRIPTION
**Pull request by isuruuy429**
## Purpose
Add tests and relevant artifacts for 1.6.4.1. and 1.6.4.2.

### CODE REVIEW

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import org.wso2.carbon.esb.scenario.test.common.http.HTTPUtils;
+import org.wso2.carbon.esb.scenario.test.common.ScenarioTestBase;
+import org.wso2.carbon.esb.scenario.test.common.http.SOAPClient;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.xml.stream.XMLStreamException;
+
+public class RemoveElementsTest extends ScenarioTestBase {
```
Comment (milindaperera) [inline product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java]: Let's add class level comment describing the high level test

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    //This test is to verify if payload can be modified by removing an element using script mediator.
+    @Test(description = "1.6.4.1-Modify-payload-by-removing-elements-using-script-mediator")
+    public void RemoveElementsUsingScriptMediator() throws IOException, XMLStreamException {
+        String url = getProxyServiceURLHttp("1_6_4_1_Proxy_RemoveElements_ScriptMediator");
+
+        String request = "<soapenv:Envelope xmlns:soapenv=\"http://schemas.xmlsoap.org/soap/envelope/\">\n"
+                + "    <soapenv:Header />\n" + "    <soapenv:Body>\n" + "        <company>WSO2</company>\n"
+                + "        <first_name>Jay</first_name>\n" + "        <last_name>Cleark</last_name>\n"
+                + "    </soapenv:Body>\n" + "</soapenv:Envelope>";
```
Comment (milindaperera) [inline product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java]: Shall we format in readable manner

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java (hunk 1 above)
Comment (isuruuy429) [reply product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java]: Added.

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java (hunk 2 above)
Comment (isuruuy429) [reply product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java]: Done

**Pull request merged**